### PR TITLE
Address funding

### PR DESCRIPTION
**Pull request by loftusa**
Partially address some issues with this page.
Also deleted the conduct page in this PR

### CODE REVIEW

File: content/about/funding.md [hunk 1]
```diff
@@ -3,18 +3,18 @@ $title@: Funding
 $order: 7
 ---
 
-This page lists currently active funding for our projects
+<img style="float: right;" src="https://bit.ly/2XPyew9" width=200 height=200>
```
Comment (falkben) [inline content/about/funding.md]: styling should go in the styles.css file unless there's specific reason not to.  just give this a class name and style it in there.

we should be hosting this image the logo directly from our site, not pulling it in from github using a url shortener

File: partials/nav.html [hunk 2]
```diff
@@ -11,6 +11,7 @@ <h4>NeuroData</h4>
         {% include 'partials/nav_content.html' %}
 
         <ul class="bottom">
+            <li><a href="https://bitsandbrains.io/2018/09/02/how-to-email-faculty-for-job.html" target="_blank"><h5>Join Us</h5></a></li>
```
Comment (falkben) [inline partials/nav.html:14]: this is fine for now, but we should make an issue to collapse the join us and contact to just a single line with icons (also adding the social media icons and links out to each)

File: content/about/funding.md [hunk 3]
```diff
@@ -3,18 +3,18 @@ $title@: Funding
...
 
-#### Synaptomes of Mouse and Man
+#### [Synaptomes of Mouse and Man](http://grantome.com/grant/NIH/R01-NS092474-01)
```
Comment (falkben) [inline content/about/funding.md]: this grant is now over and should remove?

**Review by falkben — CHANGES_REQUESTED** [review]

File: content/about/funding.md (hunk 1 above)
Comment (loftusa) [reply content/about/funding.md]: ok, sounds good. I don't immediately know how to do this, so I'm going to resolve this next monday, if that's ok with you.

File: partials/nav.html (hunk 2 above)
Comment (loftusa) [reply partials/nav.html:14]: issue made

File: content/about/funding.md (hunk 3 above)
Comment (loftusa) [reply content/about/funding.md]: good catch, done

File: content/about/funding.md (hunk 1 above)
Comment (falkben) [reply content/about/funding.md]: can you delete, then we can merge the other updates in this PR to the funding page?

Comment (loftusa) [reply content/about/funding.md]: Sure, done

**Review by falkben — APPROVED** [review]

**Pull request merged**